src/sonic-config-engine/minigraph.py

```diff
@@ -1090,6 +1092,7 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
     port_speeds_default = {}
     port_speed_png = {}
     port_descriptions = {}
+    port_alias_list = []
```

Comment:
Do we need to create a new list or can we use existing `port_alias_map`?.

Reply:
`port_alias_map` contains aliases from config_db, so we cannot use it